### ISSUE TITLE
Problem with .3ds files

### ISSUE THREAD

**Issue by tsisarv** [Asia81/HackingToolkit9DS#19]
I have a problem with decrypting .3ds files. In console(in program) i enter "d" to extract .3ds file, then enter correct name of file, then after five seconds the program says that it has extracted the .3ds file, but no files  appears in the folder.
This problem is with "bella sara the magic horse adventures" game.
Also tried to "run as admin", but it didn`t work either.
I can send you .3ds encrypted file to email if you want(it`s only 125mb game).
Thanks in advance

**Comment by tsisarv** [comment]
Sorry, it seems that my cmd doesn`t work(don`t run commands properly). Do you have any suggestions how to correct it?

**Comment by Asia81** [comment]
the latest release work only with encrypted files
Then what happen if you run each of there programs?
3dstool.exe
ctrtool.exe
makerom.exe

It looks like a tool can not run properly on your computer if no file is extracted.

**Comment by tsisarv** [comment]
Well, my cmd works fine(tried with other programs). I`ve found the decrypted version of this game on the internet, so I no longer need decryption.
"Then what happen if you run each of there programs?
3dstool.exe
ctrtool.exe
makerom.exe"
 It writes only that it`s done(very quickly, because it`s not doing anything really). It shows messages as usual, as if everything works properly, but it doesn`t. 
I`ve also deleted encrypted game, but i can redownload it and send to you, if you wanna try it yourself.

A also have another problem, though. I`ve unpacked game files, but they are in some unusual .lz format(no one .lz extractor can understand it!). The game developer used it`s own engine (not unity or unreal), so it`s hard for me to guess which program to use to see the original 3d models and textures from game(in cytra it works fine).

**Comment by Asia81** [comment]
Wait, I don't understand what you say, you managed to extract your game, yes or no.
And sorry but I'm not gonna help you on .lz things, this is not the purpose of this tool, ask .lz extractor's developper instead.

**Comment by tsisarv** [comment]
NO, i didn`t manage to extract and decrypt the game from crypted ROM. I`ve downloaded decrypted ROM and extracted from it(using another tool, not yours). Your tools didn`t worked properly for this game(i mean crypted ROM)

**Comment by Vaatuu** [comment]
Hey, Not seeing a resolution here, but I have exactly the same issue (Initial one up top)
I enter "d" to extract .3ds file, then enter correct name of file, then after five seconds the program says that it has extracted the .3ds file, but no files appears in the folder.
 - I went ahead to test it with a .CXI file, from where the files DOES appear in the folder, so obviously Im just doing something wrong.
Please help?
